docker-compose.yml

```diff
@@ -10,11 +10,12 @@ services:
       FLEXION_PRIVATE_KEY_NAME: trusted-intermediary-private-key-local.pem #pragma: allowlist secret
       FLEXION_CLIENT_NAME: flexion.simulated-hospital
       QUEUE_MAX_DELIVERY_ATTEMPTS: 5
-      SFTP_USER: ti_user
-      SFTP_PASSWORD: ti_password #pragma: allowlist secret
+      SFTP_USER_NAME: sftp_user.txt
```

Comment:
Does each variable need to be in a separate file?

Reply:
Yes, because of how the local secrets manager works - it's analogous to the Azure one, where there's a separate resource for each secret